src/Umbraco.Web.Common/AspNetCore/UmbracoViewPage.cs

```diff
@@ -68,15 +61,15 @@ public override void WriteLiteral(object value)
                         {
                             // creating previewBadge markup
                             markupToInject =
-                                string.Format(_contentSettings.PreviewBadge,
-                                    Current.IOHelper.ResolveUrl(_globalSettings.UmbracoPath),
+                                string.Format(ContentSettings.PreviewBadge,
+                                    Current.IOHelper.ResolveUrl(GlobalSettings.UmbracoPath),
```

Comment:
Should this `Current.IOHelper` be removed and replace with a private class variable, set from `Context.RequestServices.GetRequiredService<IIOHelper>()`?  Just to follow the same pattern as the use of `IGlobalSettings` etc.?